lib/oxidized/model/tplink.rb

```diff
@@ -34,27 +40,39 @@ class TPLink < Oxidized::Model
     cfg
   end
 
-  cmd 'show system-info' do |cfg|
+  cmd "show system-info" do |cfg|
     comment cfg.each_line.to_a[3..-3].join
   end
 
-  cmd 'show running-config' do |cfg|
+  cmd "show running-config" do |cfg|
     lines = cfg.each_line.to_a[1..-1]
     # cut config after "end"
     lines[0..lines.index("end\n")].join
```

Comment:
For my T1700G-28TQRev2 with V3 there is no end it just quits. Dunno how relevant it is tho :)  

```
 ip igmp snooping vlan-config 3004 ltime 1
 ip igmp snooping vlan-config 3004 querier
 ip igmp snooping  vlan-config 3004 querier general-query source-ip 172.16.4.1
#

sw2#

sw2#

sw2#
```

Reply:
I guess it isn't a problem. Problem is to have the `end` and save the garbage after it in the backup.